### ISSUE TITLE
load_learner: 'method-wrapper' object has no attribute '__code__'

### ISSUE THREAD

**Issue by odysseus0** [fastai/fastai#1777]
**Describe the bug**
load_learner: 'method-wrapper' object has no attribute '__code__'

**Provide your installation details**

```text
=== Software === 
python        : 3.7.1
fastai        : 1.0.47
fastprogress  : 0.1.20
torch         : 1.0.1.post2
nvidia driver : 410.72
torch cuda    : 10.0.130 / is available
torch cudnn   : 7402 / is enabled

=== Hardware === 
nvidia gpus   : 1
torch devices : 1
  - gpu0      : 16280MB | Tesla P100-PCIE-16GB

=== Environment === 
platform      : Linux-4.9.0-8-amd64-x86_64-with-debian-9.8
distro        : #1 SMP Debian 4.9.130-2 (2018-10-27)
conda env     : base
python        : /opt/anaconda3/bin/python
sys.path      : /home/jupyter/MURA
/opt/anaconda3/lib/python37.zip
/opt/anaconda3/lib/python3.7
/opt/anaconda3/lib/python3.7/lib-dynload

/opt/anaconda3/lib/python3.7/site-packages
/opt/anaconda3/lib/python3.7/site-packages/IPython/extensions
/home/jupyter/.ipython
```

**To Reproduce**

**Stack Trace**

```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/opt/anaconda3/lib/python3.7/site-packages/IPython/core/formatters.py in __call__(self, obj)
    700                 type_pprinters=self.type_printers,
    701                 deferred_pprinters=self.deferred_printers)
--> 702             printer.pretty(obj)
    703             printer.flush()
    704             return stream.getvalue()

/opt/anaconda3/lib/python3.7/site-packages/IPython/lib/pretty.py in pretty(self, obj)
    400                         if cls is not object \
    401                                 and callable(cls.__dict__.get('__repr__')):
--> 402                             return _repr_pprint(obj, self, cycle)
    403 
    404             return _default_pprint(obj, self, cycle)

/opt/anaconda3/lib/python3.7/site-packages/IPython/lib/pretty.py in _repr_pprint(obj, p, cycle)
    695     """A pprint that just redirects to the normal repr function."""
    696     # Find newlines and replace them with p.break_()
--> 697     output = repr(obj)
    698     for idx,output_line in enumerate(output.splitlines()):
    699         if idx:

/opt/anaconda3/lib/python3.7/dataclasses.py in __repr__(self)

/opt/anaconda3/lib/python3.7/site-packages/fastai/callback.py in __repr__(self)
    176 
    177     def  __repr__(self):
--> 178         attrs = func_args(self.__init__)
    179         to_remove = getattr(self, 'exclude', [])
    180         list_repr = [self.__class__.__name__] + [f'{k}: {getattr(self, k)}' for k in attrs if k != 'self' and k not in to_remove]

/opt/anaconda3/lib/python3.7/site-packages/fastai/core.py in func_args(func)
    246 def func_args(func)->bool:
    247     "Return the arguments of `func`."
--> 248     code = func.__code__
    249     return code.co_varnames[:code.co_argcount]
    250 

AttributeError: 'method-wrapper' object has no attribute '__code__'
```

**Comment by maykulkarni** [comment]
Had the same [issue](https://github.com/fastai/fastai/issues/1767) yesterday. Add a `;` at the end of the `.load` method to fix the problem

**Comment by stas00** [comment]
If it's the same, it was fixed a few hours ago by @sgugger:
https://github.com/fastai/fastai/commit/d5c8be72d6fe14bac7933dfda0b1786442f0aa94
Please try the git master and let us know whether it's resolved for you too.
The fix will be out in the next release.

**Comment by sgugger** [comment]
And please don't forget to give us your code or a reproducible example when filing an issue, otherwise we can't solve it.

**Comment by odysseus0** [comment]
It is very likely the same issue. I am also using cohen's kappa coefficient. Let me try again.

**Comment by sgugger** [comment]
Closing then, please re-open if needed.
